generator/osm2type.cpp

```diff
@@ -138,6 +138,11 @@ namespace ftype
         ++token;
         lang = (token ? *token : "default");
 
+        if (++token) {
+          // Do not consider languages with suffixes, like "en:pronunciation".
+          lang = "<unknown>";
```

Comment:
Why not "default"?

Reply:
Because `default` means the default name, contents of a `name` tag. I deliberately set a value that is guaranteed not to be present in the language list.

Reply:
Turns out I could simply do `return false`. PTAL.